File: tests/fixtures/python/correct_physics.py
"""Test fixture: correct physics code with no violations.

Benchmark 1.5 from AUDIT_PLAN.md.
Expected result: Zero violations. All units correctly inferred.
"""

distance = 100.0    # unit: m
time = 10.0         # unit: s
velocity = distance / time              # inferred: m/s (correct)
acceleration = velocity / time          # inferred: m/s^2 (correct)

File: tests/fixtures/fortran/correct_physics.f90
! Test fixture: correct physics computation.
!
! Computes velocity and acceleration from distance and time.
! Expected result: zero violations.

program correct_physics
  implicit none
  real :: distance  ! unit: m
  real :: time_val  ! unit: s
  real :: velocity
  real :: acceleration

  distance = 100.0
  time_val = 10.0
  velocity = distance / time_val
  acceleration = velocity / time_val
end program correct_physics
